### ISSUE TITLE
Use make to always get latest dependent libs

### ISSUE THREAD

**Issue by marcoarthur** [subutai-io/gorjun#198]
Write dependency rules to modules, so we guarantee we build gorjun against the latest go modules. Since this is the rule for Jenkins build

**Comment by marcoarthur** [comment]
make should follow same approach used to build gorjun in Jenkins. This way we garantee that won't crash when official building. Reference:

https://github.com/subutai-io/gorjun/blob/dev/Jenkinsfile

**Comment by crioto** [comment]
@marcoarthur basically single `go get -u` execution on dev's host should bring you latest libraries. This is not used in jenkins file due to it's go environment. `go get -u` just won't work on jenkins.

**Comment by marcoarthur** [comment]
Yeah I think so, but the problem here is that go get -u takes too long since that potentially could update all system. Better approach is needed

**Comment by Dilshat** [comment]
Look at **vendoring** in go dependency management
